contribution/testing_guide.md

```diff
@@ -56,28 +56,25 @@ $ kubectl -n [namespace name] exec -it [pod name] -- bash -c [command]
 
 ## 1.6. Check generated alerts
 
-- Log file
+- Log using [karmor](https://github.com/kubearmor/kubearmor-client) cli tool
 
     ```text
-    $ tail (-f) /tmp/kubearmor.log
+    $ karmor log [flags]
     ```
-
-    If you changed the location of a log file, replace the default file path to your file.
-
-    ```text
-    $ tail (-f) [your log file path]
-    ```
-
-- Log client
-
-    Compile a log client
+    
+    flags:
 
     ```text
-    $ git clone https://github.com/kubearmor/kubearmor-log-client
-    $ cd kubearmor-log-client
-    ~/kubearmor-log-client$ make
+        --gRPC string        gRPC server information
+    -h, --help               help for log
+        --json               Flag to print alerts and logs in the JSON format
+        --logFilter string   Filter for what kinds of alerts and logs to receive, {policy|system|all} (default "policy")
+        --logPath string     Output location for alerts and logs, {path|stdout|none} (default "stdout")
+        --msgPath string     Output location for messages, {path|stdout|none} (default "none")
     ```
-
+    
+    Note: Messages, alerts, and logs will be generated right after the karmor runs logs, to avoid any kind of interference, run the above command in other terminal to see logs live and in an interactive way.
+    
     Run the log client
```

Comment:
Let's remove this since as I mentioned kArmor solves the purpose, we don't need to mention about log client.

Reply:
ok,